duet/views/mobile_ios_country.view.lkml

```diff
@@ -0,0 +1,368 @@
+            THEN
+              'Russian Federation'
+            WHEN
+              storefront = 'Tanzania'
+            THEN
+              'Tanzania, United Republic of'
+            WHEN
+              storefront = 'North Macedonia'
+            THEN
+              'Macedonia, the Former Yugoslav Republic of'
+            WHEN
+              storefront = 'São Tomé and Príncipe'
+            THEN
+              'Sao Tome and Principe'
+            WHEN
+              storefront = 'Micronesia'
+            THEN
+              'Micronesia, Federated States of'
+            WHEN
+              storefront = 'Moldova'
+            THEN
+              'Moldova, Republic of'
+            WHEN
+              storefront = 'Venezuela'
+            THEN
+              'Venezuela, Bolivarian Republic of'
+            WHEN
+              storefront = 'Taiwan'
+            THEN
+              'Taiwan, Province of China'
+            WHEN
+              storefront = 'British Virgin Islands'
+            THEN
+              'Virgin Islands, British'
+            WHEN
+              storefront = 'Brunei'
+            THEN
+              'Brunei Darussalam'
+            WHEN
+              storefront = 'Congo, Democratic Republic of the'
+            THEN
+              'Congo, the Democratic Republic of the'
+            WHEN
+              storefront = 'Congo, Republic of the'
+            THEN
+              'Congo'
+            WHEN
+              storefront = 'Bolivia'
+            THEN
+              'Bolivia, Plurinational State of'
+            WHEN
+              storefront = 'Vietnam'
+            THEN
+              'Viet Nam'
+            WHEN
+              storefront = "Cote d'Ivoire"
+            THEN
+              "Côte d'Ivoire"
+            ELSE
+              storefront
+            END
+            AS name,
+            product_page_views,
+            app_units AS first_time_installs,
+            installations_opt_in
+          FROM
+            `moz-fx-data-marketing-prod.apple_app_store.metrics_by_storefront`
+          CROSS JOIN
+            period
+          WHERE
+            date
+            BETWEEN start_date
+            AND end_date
+            AND app_name IN (
+              {% if app_id._parameter_value == "firefox" %} "Firefox"
+              {% elsif app_id._parameter_value == "focus" %} "Focus"
+              {% elsif app_id._parameter_value == "klar" %} "Klar"
+              {% endif %}
+            )
+        ),
+        apple_country_metrics AS (
+          SELECT
+            date AS submission_date,
+            app_name,
+           -- Macau, Eswatini, Laos
+            CASE
+            WHEN
+              name = 'Macau'
+            THEN
+              'MO'
+            WHEN
+              name = 'Eswatini'
+            THEN
+              'SZ'
+            WHEN
+              name = 'Laos'
+            THEN
+              'LA'
+            ELSE
+              code
+            END
+            AS country,
```

Comment:
That static dataset doesn't have these?

Reply:
The apple store doesn't have country code, only country names (for the store front). They also use different names in a few dozen cases that have to be manually mapped e.g. China Mainland. This is mapping the country name to the static dataset.

I believe those countries are missing from the static dataset, although @ksiegler1 worked through this logic for the most part.

Reply:
Yes these ones had to be manually added because I could not locate them in the static country codes dataset.